app/models/big_query_client.rb

```diff
@@ -19,4 +19,14 @@ def initialize
     self.client = Google::Cloud::Bigquery.new
   end
 
+  # clears the entire BQ table.  This is only intended for utility use in tests/development
+  # it will no-op in production
+  def self.clear_bq_table
+    if Rails.env.production?
```

Comment:
I might invert this and move the delete call into the block that checks if the environment is `test` or `development`.  I can't see us wanting to do this on `staging` or other deployed instances.

Reply:
Interesting -- I was actually thinking it might be useful if we ever decide to clean out/reset staging.  I could see this going either way, and since deleting the BQ table on staging wouldn't be that big of a deal (we'd just need to reparse the metadata files for the convention studies), I'll leave it